src/lib/components/table/components/TableCell/ButtonCell/ButtonCell.module.scss

```diff
@@ -18,10 +18,6 @@
       background-color: $ds-grey-200;
     }
   }
-
-  @include p-size-mobile {
-    pointer-events: none;
```

Comment:
This was intentionally added, can you explain why we need to remove this?

In the design's I have seen so far, ButtonCell was only ever used as a "selector button" for selecting a tariff or something. On desktop it's fine to let the button itself trigger this. On mobile however, the selection is triggered via the arrow keys, not the "button" in the center. Letting the button in the middle behave like a button in those cases lead to weird edge cases.

Reply:
Oh, I see, I was trying to click the button but it wasn't working, I didn't realise I had to manually set this up using `onSelectionChanged`. I'll close this PR and update my code to use it instead.